data/coalition/coalition missions.txt

```diff
@@ -1392,3 +1392,207 @@ mission "Arachi Cartoonist 2"
+	on complete
+		conversation
+			branch drama
+				has "Coalition: Council of Ahr: Drama: offered"
+			`"Captain <last>! A pleasurable experience for the both of us, this has been, but sadly short, the trip from my home to <planet> is, so part ways, we must," Plubat says as he prepares to leave your ship. "To finish the tour of our facilities, I'm sure you want, so once over, the Council is, ask for me there, and guide you again, I shall."`
+				accept
+			label drama
+			`House Chamgar's representative, Plubat, stays close to you while firmly clinging to his luggage, as his colleage from house Bebliss, Lady Plubat, has her bags brought out of your ship by some nearby Arachi in uniform. "Telling you I am, Captain, trusted, she should not be. Checked your ship, have you? All in place, your belongings are? Careful with her types, you must be," he says. After the Bebliss butlers are done, Plubat scoffs something at Lady Plubat one last time, then leaves your ship.`
+
+mission "Coalition: Council of Ahr: Drama"
+	invisible
+	landing
+	to offer
+		or
+			has "Coalition: Council of Ahr: Bebliss: active"
+			has "Coalition: Council of Ahr: Chamgar: active"
+		has "Coalition: Council of Ahr: active"
+	source
+		planet "Hammer of Debrugt" "Stronghold of Flugbu"
+	destination "Ahr"
+	on enter
+		conversation
+			`After departing from <origin>, you start hearing some shouting from outside your cockpit, so you leave to check up on your passengers. Plubat, from House Chamgar, is banging one leg against the door to the room of Plubat, from House Bebliss. "Captain! Good thing that here, you are," he says. "Sent an envoy to mock me, that damned house Bebliss has! Copying my name, she is!"`
+			`	Plubat, from House Bebliss, opens the door to her room to respond. "Nonsense that is, Captain. Look at us, several decades older, I am, so copying you, how could I be? The offending party here, house Chamgar is."`
+			`	"Changed your name for this mockery, you could have. Know of your house's grudge against us, I do," he replies.`
+			`	"Absurd, childish, that is. The one to start mocking me, you were," she says. "Captain <last>, urge you to make haste for Ahr, I must. Put up with this buffoon for much longer, I cannot."`
+			`	Lady Plubat closes the door to her room again, and you convince house Chamgar's representative to also go back to his room. You return to your cockpit, hoping that's the last of that.`
+	on enter "Sol Arach"
+		dialog
+			`In spite of your best attempts to keep them away from each other, Plubat and Plubat have been brief on their breaks from insulting one another. However, they head back to their rooms on their own once you enter this system, without you needing to leave the cockpit to interfere. Being so close to their destination, you hope they won't give you any more headaches.`
+
+mission "Coalition: Council of Ahr: On Complete"
```

Comment:
Any reason why this isn't in the `on complete` of `Coalition: Council of Ahr` and instead its own separate mission? Is it related to that discussion you had in Discord about the order in which the mission completion conversations show?

Reply:
Yes, when I had it as the `on complete` for the main mission, it would always pop up before the others when you completed it. This is so that it always pops up after them.